### DISCUSSION TITLE
Using the model on baseten the results are significantly worse/utterly different than what the paper claims, why?

### DISCUSSION THREAD

**Discussion by papajo-r** [TencentARC/GFPGAN#225] [General]
Why are the results on https://app.baseten.co/apps/QPp4nPE/operator_views/RqgOnqV significantly worse that what the paper claims? In fact https://app.baseten.co/apps/QPp4nPE/operator_views/RqgOnqV does nothing other than increasing the the sharpness and adding some textures based on nothing real (e.g connect a missing peace with similar pixels to the pieces that are around it)  

Not a single one of my photos "worked" they all still had creases or scratches and all the faces were worse than before just with some extra pixels...

I also used the paper's examples and these came out worse than what the paper claimed as a result e.g

![image](https://user-images.githubusercontent.com/18315987/182016069-04756e98-22e2-416c-8cb4-b2c52f134094.png)
and 
![palio 2](https://user-images.githubusercontent.com/18315987/182016088-5db258a3-e7c2-42d7-9ae6-6987a23c3f6f.png)
 
 Which are day and night different (the above are the "RESTORED" results by https://app.baseten.co/apps/QPp4nPE/operator_views/RqgOnqV and NOT the sources! )  compared to what the paper claims to have been achieved, I also tried to do a multiple run of the restored version (used eached restored version as a source and tried again) but this looked even worse 
 e.g 
![palio 10](https://user-images.githubusercontent.com/18315987/182016170-ab66abec-8d8d-48ec-9fcd-2300685ccaca.png)

**Comment by xeophyte** [comment]
The link (demo) doesn't work anymore?
